rolling/distribution.yaml

```diff
@@ -2023,6 +2023,21 @@ repositories:
       url: https://github.com/ros/resource_retriever.git
       version: ros2
     status: maintained
+  rmf_battery:
+    doc:
+      type: git
+      url: https://github.com/open-rmf/rmf_battery.git
+      version: rolling
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/rmf_battery-release.git
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/open-rmf/rmf_battery.git
+      version: 0.1.0
```

Comment:
I think the source entry version is expected to be the development branch. Tools like `rosinstall_generator` use this info to checkout the upstream development version.

Reply:
Ok, just changed it to `main`, hope it's fine now.